### ISSUE TITLE
Stolen Content

### ISSUE THREAD

**Issue by Mixter213** [MrRobotjs/BetterDocs#195]
I would like to see the permission you received to use any of these themes you are claiming and posting on your knock-off of a website.

**Comment by PlusInsta** [comment]
BetterDocs just links to other people's themes though. It's really more of a database than a "knock-off". The website credits the author of the themes and point to their repository for downloads. They aren't being "claimed" by anyone, nor are the themes "stolen". If they really were stolen, the authors wouldn't get any credit.

It's surprising to me that my theme got featured on the website without me knowing, but they're still not doing anything wrong here. As a theme author, I'll gladly take the exposure. If I really wanted to, I could even request my theme to be taken off the website.

But then again...[I have a feeling you're not exactly fond of BetterDocs](https://github.com/Mixter213/Discord-Themes/commit/a243045024a148dfa380b0c287a875c710c996cb).

**Comment by Mixter213** [comment]
Better docs does not care about the authors input on whether or not they want their theme or plugin on the shitty website. I know many authors including myself who do not want and requested for the removal of said themes. and was ignored. aswell as it's not really a database when the first thing you see for themes is "Start customizing your Discordapp with our best themes". this isn't just to be a data base. this is falsely saying shit docs is the main organizer of themes which is far far from the truth. If you want a supported theme or plugin you should go to BetterDiscord discord server. not this shit website.

**Comment by PlusInsta** [comment]
Clearly you're misunderstanding something. 
This website points to your theme's repo directly. It credits you as the author.
You're in full control over whether or not your theme is featured on BetterDocs.

If you want to have your theme removed, all you have to do is **make a pull request that blanks out the theme's file.** [The repository you're looking for is right here.](https://github.com/MrRobotjs/BetterDocs-React/tree/master/src/themes)
Again, edit your theme's file to be completely blank, then make a pull request. It's that simple.

Please, for the sake of everyone on GitHub, do not whine about things you can fix yourself. If you had made a pull request removing the theme file, @MrRobotjs would already have merged it.

**Comment by Mixter213** [comment]
If it is that simple i will leave it at that. but I nor anyone else in the bd community wants or desires their theme in shit docs. an opt-out option does not mean you chose it to be there. and you seem to of missed the part where shit docs is claiming the themes as its community by stating "our themes"

**Comment by PlusInsta** [comment]
By "our themes", he's likely referring to everyone who makes themes & plugins as a community of Discord modders. You're taking things too literally.

**Comment by Mixter213** [comment]
nah, im not taking it to seriously. well atleast anything downloaded from shit docs thats not in theme-repo or modified from it won't get support from the theme dev in better discord server. in fact the phrase better docs is banned. also to early to tell so far, but my theme still has yet to be taken down

**Comment by Mixter213** [comment]
Just as an update - He merged the pull requests on this repository but as i learned very quickly this is an outdated one. so i made one to remove my theme from the updated repository. He has closed (not merged) one and so far ignored the other. aswell as closed (not merged) another users request to remove his themes. So far its a lot less of what you say
